frontend/src/components/Map/Map.scss

```diff
@@ -0,0 +1,52 @@
+.Map {
+  position: relative;
+  width: 100%;
+  height: 100%;
+
+  .gm-svpc,
+  .gmnoprint {
+    display: none;
+  }
+}
+
+.Map-infoWrapper {
+  position: absolute;
+  left: -10rem;
+  top: 4rem;
+  width: 320px;
+  height: 230px;
+  transform: scale(.8);
+  background: white;
+  border: 1px solid #ccc;
+  border-radius: 10px;
+  opacity: 0;
+  transition: opacity .2s ease-in-out,
+            transform .15s ease-in-out;
+
+  &--visible {
+    transform: scale(1);
+    opacity: 1;
+  }
+
+  h1 {
+    font-size: 20px;
+    font-weight: 600;
+  }
+
+  p {
+    margin-bottom: 10px;
+    font-size: 14px;
+  }
```

Comment:
This should actually be the style for all the paragraphs. That's why I initially set the `body` font-size to 14px.

Reply:
Got it!

Reply:
It's currently falling back to the default (16px). I'll set it to 14px.

Reply:
Also the p's in the map component are inheriting styles from the lib so i need to set a font-size here.